specs/Storage.Pickers/FileSavePicker.md

```diff
@@ -23,12 +23,26 @@ runtimeclass FileSavePicker
 
     IMap<string, IVector<string>> FileTypeChoices{ get; };
 
+    String SuggestedStartFolder;
     PickerLocationId SuggestedStartLocation;
 
     Windows.Foundation.IAsyncOperation<PickFileResult> PickSaveFileAsync();
 }
 ```
 
+Notes: **Understanding SuggestedStartFolder/SuggestedStartLocation vs SuggestedFolder:**
+
+These two properties have fundamentally different behaviors in terms of when and how they affect the picker:
```

Comment:
>These [](http://example.com/codeflow?start=0&length=5)

Add line breaks to wrap these lines at a reasonable width to improve readability.